app/device/gyro.py

```diff
@@ -1,3 +1,63 @@
-class Gyro(object):
-    def __init__(self):
-        print 'lol'
+import smbus
+
+class Gyro:
+    'class that represents a gyroscope on a rocket'
+
+    itg_add = (0x68) # ITG3200 address
```

Comment:
these can be constants writtent in caps and without the brackets (ex. `ITG_ADD = 0x68`)